modules/adding-ibmz-lpar-agent.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="adding-ibmz-lpar-agents_{context}"]
+= Adding {ibm-z-title} Logical Partition (LPAR) as agents
```

Comment:
How is this getting populated? {ibm-z-title}

Reply:
<img width="782" alt="image" src="https://github.com/user-attachments/assets/e045173e-ff64-471b-87a4-f8657d26d9f0">
Like this